docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+
+Optional arguments
+"""""""""""""""""""
+Four of the optional arguments can be used to narrow down our search results:
+
+.. code-block:: none
+
+  -c | --cloudcover
+  minimum,maximum
+  The cloud cover range must be specified in %
+  Default: 0,100
+
+  -d | --daterange
+  starttime,endtime
+  Dates must be given in the following format: YYYYMMDD,YYYYMMDD
+  Default: 19700101,today
+
+  -s | --sensor
+  Sensors to include in the query, comma-separated.
+  Valid sensors:
+  Landsat                             Sentinel
+  LT04 - Landsat 4 TM                 S2A - Sentinel-2A MSI
+  LT05 - Landsat 5 TM                 S2B - Sentinel-2B MSI
+  LE07 - Landsat 7 ETM+
+  LC08 - Landsat 8 OLI
+  Default: LT04,LT05,LE07,LC08,S2A,S2B
+
+  -t | --tier
+  Landsat collection tier level. Valid tiers: T1,T2,RT
+  Default: T1
+
+We can filter by cloud cover estimate, date range, sensor, and tier (Landsat only). If these parameters aren't set, they will default to the values above. Note that the default search is a very broad one. It will basically return all Landsat **and** Sentinel-2 scenes ever acquired for the area of interest. The only exception is the Tier level for Landsat - only Tier 1 scenes will be included, as Tier 2 and Real Time scenes are not desirable for most applications.
+
+There are two further optional arguments that don't affect the search parameters, but change the behavior of FORCE Level 1 CSD:
+
+.. code-block:: none
+
+  -n | --no-act
+  Will trigger a dry run that will only return the number of images
+  and their total data volume
+
+  -k | --keep-meta
+  Will write the results of the query to the level 1 datapool directory.
+  Two files will be created if Landsat and Sentinel-2 data is queried
+  at the same time. Filename: csd_metadata_YYYY-MM-DDTHH-MM-SS
+
+
+**Please note** that the mandatory arguments are positional! They need to be provided in this exact order. The optional arguments can be placed anywhere and may also be combined. For example, ``-n -k -c 0,70`` could also be written as ``-nkc 0,70``. When values are passed to the optional arguments (cloud cover, date range, sensor, or tier), these must be separated by commas ``,`` and must not contain whitespace.
+
+Querying and downloading data
+-----------------------------
+
+Basic search
+""""""""""""
+
+Now that the FORCE Level 1 CSD is set up and we got the basic syntax straight, we can start looking for scenes that match our needs. Let's start with a very broad search using a geopackage file defining our area of interest (the city-state of Berlin, Germany) as polygon feature. You may also use point or line features as aois.
+
+To get a first overview of the data volume, we will use the ``-n | --no-act`` option. This will print information about the results of our search without actually downloading the data.
+
+
+.. code-block:: none
+
+    force-level1-csd meta gcs_berlin/level1 gcs_berlin/level1/l1_pool.txt gcs_berlin/berlin_aoi.gpkg -n
```

Comment:
Not that it matters, but good practice would still be to put the optional arguments first